invenio_communities/communities/services/config.py

```diff
@@ -111,6 +111,8 @@ class CommunityServiceConfig(RecordServiceConfig):
         "{+api}/communities/{community_id}/requests{?args*}"
     )
 
+    community_logo_size_default_quota = 10**6
```

Comment:
Does it make sense to have a default value? I put it here to avoid issues if the app config is missing.

Reply:
I did not understand the suggestion sorry. 

Should I move this variable outside the config file?